fretwell-distributed-government.md

```diff
@@ -1,98 +1,93 @@
 # Distributed government is the path to a truly open, empathetic, inclusive civil society
 Luke Fretwell
 
-Society has reached a point in the organizational and civic evolutionary cycle where [distributed government teams](https://distributedgov.com/) will play an undeniable and critical role in the future of highly effective, empathetic and inclusive public services.
+Society has reached a point in the organizational and civic evolutionary cycle where [distributed government teams](https://distributedgov.com/) will play an undeniable and critical role in the future of highly effective, empathetic, and inclusive public services.
 
-As civic leaders wrestle with and seriously address the issues of our times — disasters, pandemics, climate change, health and wellness, economic empowerment — distributed teams is the obvious solution for delivering responsive and resilient government services.
+As civic leaders wrestle with and seriously address the issues of our times—disasters, pandemics, climate change, health and wellness, economic empowerment—operating through distributed teams is the obvious solution for delivering responsive and resilient government services.
 
-## Defining distributed
-
-John O'Duinn, author of[ Distributed Teams: The Art and Practice of Working Together While Physically Apart](https://www.amazon.com/dp/1732254907/ref=cm_sw_su_dp), has written the definitive book on why distributed teams are important, and how they can be even more effective than their physically collocated counterparts.
-
-While antiquated terms like 'telework' and 'remote' continue to be flippantly used, O'Duinn makes clear [distinctions](https://oduinn.com/2020/02/25/distributed-team-vs-remote-work-and-work-from-home/) between key terms, such as 'distributed teams,' 'virtual teams,' 'virtual employee,' 'remote work,' 'remote employee,' 'work from home,' 'work from anywhere' and 'telework'.
-
-And the subtle differences in each drastically impact our perceptions and assumptions around distributed teams.
-
-How O'Duinn describes 'distributed':
-
-All humans on the team work together, even though they are physically apart from each other. This is not a collection of individuals who each do solo heads-down work from different locations. Instead, this is a group of humans who coordinate their work with others on their physically distributed team. Because everyone on the physically distributed team is "remote" from someone, it is clear that everyone on the team has equal responsibility to communicate and coordinate their work with coworkers – regardless of whether any individual human is working from a building with the company logo on the door, from home, from a coworking space, a hotel or a parked car! Example usage: “I work on a distributed team”, “my team is distributed”.
+In this chapter ... 
```

Comment:
This is a placeholder for us to revisit when the chapter is complete. It's a reminder to add a brief "preview" of the chapter, an overview for readers, but that's not possible until we have a clear sense of the entire scope of the thing.